doc/api/module.md

```diff
@@ -608,7 +611,7 @@ changes:
 
 <!-- type=misc -->
 
-> Stability: 1.2 - Release candidate (asynchronous version)
+> Stability: 1 - Experimental (asynchronous version)
```

Comment:
Any stability level 1.x is experimental. See https://nodejs.org/docs/latest/api/documentation.html#stability-index

We don’t really have a level for “pending removal” so maybe the most appropriate level is the most experimental one:

```suggestion
> Stability: 1.0 - Early development (asynchronous version)
```

Reply:
It's not early development though

Reply:
I think deprecated would be the state for pending removal but that seems too early to me.

I selected a similar state/wording as for [async_hooks](https://nodejs.org/docs/latest/api/async_hooks.html) (except the migration hints) because it seems very similar:
* no 100% replacement as of now
* works fine for some usecases
* has flows which can't/are not planned to be fixed
* high overhead at least in some cases

Reply:
It was decided that "deprecated" was reserved for stable features: https://github.com/nodejs/node/blob/38bf955937b5a30f5bd59360c2b153647d4737e2/doc/api/documentation.md?plain=1#L48-L49

Reply:
Maybe just create a new stage like `1.9 - Pending removal`? And we can add a note next to it that the sync hooks are the intended replacement.

Reply:
If neither `1.2 - Release candidate` nor `1.0 - Early Development` makes sense, maybe `1.1 - Active development` ?

I feel that whether/when to remove it would be a different question due to the existing usage, but it somewhat certain that even if it stays, it would have to have changes in order to be really functional - there are too many quirks right now.

Reply:
I agree that `1.1 - Active development` is better than 1.2, but I think just making a new `Pending removal` stage would be better still. Worst of all would be going to just `1 - Experimental`, as that doesn’t provide any guidance. Having a detailed note is good, but there’s tooling built around our docs and so I think it’s better if the most important information (that this is pending removal) is presented in a structured data way that tooling can understand, rather than contained only within a natural English note.